quinn-proto/src/crypto/rustls.rs

```diff
@@ -45,16 +43,31 @@ impl crypto::Session for TlsSession {
     type AuthenticationData = AuthenticationData;
     type ClientConfig = Arc<rustls::ClientConfig>;
     type HmacKey = hmac::Key;
-    type HeaderKey = aead::quic::HeaderProtectionKey;
     type Key = PacketKey;
+    type HeaderKey = HeaderProtectionKey;
     type ServerConfig = Arc<rustls::ServerConfig>;
 
-    /// Create the initial set of keys given the initial ConnectionId
     fn initial_keys(
-        id: &ConnectionId,
+        conn_id: &ConnectionId,
```

Comment:
I think we don't use `conn_id` much because we usually use `cid` (though I guess only in combination with a prefix like `loc_cid`, `dst_cid` or `rem_cid`), maybe it might be nicer to reuse something common here?